src/app/components/core/filter/add-filter-wizard/add-filter-wizard.component.ts

```diff
@@ -102,6 +105,15 @@ export class AddFilterWizardComponent implements OnInit {
         nxtButton.textContent = 'Next';
         previousButton.textContent = 'Previous';
 
+         // To verify if filter with given name already exist
```

Comment:
// To verify if category (or filter itself) with this name already exists hence filter can not be created with that name